util/projectRequest.ts

```diff
@@ -3,6 +3,43 @@ import { paginateRest } from '@octokit/plugin-paginate-rest';
 import githubColorsFixture from '../data/githubColors.json';
 import { Project, ACMCommitteeTopics, GitHubColors } from './types';
 
+export async function getUclaOpenSource(): Promise<Project[]> {
+  const PaginatedOctokit = Octokit.plugin(paginateRest);
+  const octokit = new PaginatedOctokit();
+  const response = await octokit.paginate('GET /search/repositories', {
+    q: 'ucla-opensource+in:topics',
+    per_page: 100,
+  });
+
+  const filteredData = response.filter((repo) => !repo.archived);
+  const sortedData = filteredData.sort(
+    (a, b) =>
+      new Date(b.updated_at as string).getTime() - new Date(a.updated_at as string).getTime(),
+  );
+  return sortedData.map((repo) =>
+    repo.homepage
+      ? {
+        name: repo.name,
+        description: repo.description ?? '',
+        link: repo.homepage ?? '',
+        repo: repo.html_url,
+        lang: repo.language ?? '',
+        topics: repo.topics ?? [],
+        image: getImageFromTopics(repo.topics).image,
```

Comment:
For our placeholder image, maybe we can use the UCLA logo instead of the ACM logo as these are projects by UCLA students? down the line, we can do stuff like grabbing the social media preview image for each project and displaying that, but that can be a stretch goal for later